arkane/isodesmicTest.py

```diff
@@ -50,13 +50,14 @@ class TestErrorCancelingReactionAndSpecies(unittest.TestCase):
     Tests that ErrorCancelingReaction objects and ErrorCancelingSpecies object are properly implemented
     """
 
-    def setUp(self):
+    @classmethod
```

Comment:
I didn't see this commit until later and I made some comments about this during my review. This should be squashed.